lib/bookingsync/engine/models/account.rb

```diff
@@ -0,0 +1,36 @@
+module BookingSync::Engine::Models::Account
+  extend ActiveSupport::Concern
+  include BookingSync::Engine::Models::BaseAccount
+
+  included do
+    validates :synced_id, uniqueness: true
+  end
+
+  module ClassMethods
+    def from_omniauth(auth, _host)
```

Comment:
why is it required if it's not used?

Reply:
Might be actually better to use `*args` here so that this argument is not necessary to pass when using it if not needed

Reply:
`from_omniauth` is used in the session controller (https://github.com/BookingSync/bookingsync-engine/pull/57/files#diff-d5241d488259f32ecbe2f636133e5ddaR4)
The class calling the method is defined dynamically hence having a specific method signature.